electron/ui/src/views/Optimization/AdvancedOptions.js

```diff
@@ -16,11 +16,11 @@ export const advancedOptions = {
         }
     },
     pipeline_cost: {
-        displayName: "Pipeline Capacity",
+        displayName: "Pipeline Cost",
         defaultValue: "distance_based",
         options: {
-            "Distance-based": "distance_based",
-            "Capacity-based": "capacity_based"
+            "Distance based": "distance_based",
+            "Capacity based": "capacity_based"
```

Comment:
```suggestion
            "Distance-based": "distance_based",
            "Capacity-based": "capacity_based"
```
Nitpick but I think it's more grammatically correct to keep the hyphens here.